_data/accessibility-tests/tooltip.yml

```
@@ -0,0 +1,96 @@
+title: Tooltip
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to communicate the purpose of the tooltip.
+    summary_additional: |
+      When you view the tooltip, the text, and never color alone, convey its purpose.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Appropriate color contrast between tooltip text and background.
+    summary_additional: |
+      When you view the tooltip and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the background and text is at least 4:5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: Appropriate color contrast between tooltip and page background.
+    summary_additional: |
+      When you view the tooltip and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the tooltip component itself and any adjacent background content is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.11
+  - summary: You can hover over tooltip content.
+    summary_additional: When you move the pointer over an active tooltip, the tooltip stays open.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
```

Hover functionality was added in `3.8.1`, we should update here

```suggestion
    version_tested: 3.8.1
```

---

Goooood catch. Updated in [1dd578e](https://github.com/uswds/uswds-site/pull/2646/commits/1dd578eb4c656dc20e8390134c91abaf513011b2)